sql/cagg_api.sql

```diff
@@ -0,0 +1,179 @@
+-- This file and its contents are licensed under the Apache License 2.0.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-APACHE for a copy of the license.
+
+-- Helper function to get information about a hypertable and the
+-- primary dimension.
+CREATE OR REPLACE FUNCTION _timescaledb_functions.get_hypertable_info(
+       hypertable REGCLASS,
+       column_type REGTYPE
```

Comment:
What is column_type? Is it the expected type of the primary dimension? 

Why not just return the info record, including the type and elide the type from the argument list?

Reply:
This is an internal function and only intended to be used for the other API functions. As such, I moved checks that are shared between the overloaded versions of `get_hypertable_invalidations` into this function to avoid duplicating code.